lib/taskana-core/src/main/java/pro/taskana/impl/TaskServiceImpl.java

```diff
@@ -773,10 +782,13 @@ private void removeNonExistingTasksFromTaskIdList(List<String> taskIds,
                 taskIdIterator.remove();
             }
         }
+        LOGGER.debug("exit from removeNonExistingTasksFromTaskIdList()");
     }
 
     private void checkIfTransferConditionsAreFulfilled(List<String> taskIds, List<MinimalTaskSummary> taskSummaries,
         BulkOperationResults<String, TaskanaException> bulkLog) {
+        LOGGER.debug("entry to checkIfTransferConditionsAreFulfilled(taskIds = {}, taskSummaries = {}, bulkLog = {})",
+            LoggerUtils.listToString(taskIds), LoggerUtils.listToString(taskSummaries), bulkLog);
         Set<String> workbasketIds = new HashSet<>();
```

Comment:
The LOGGER statement on line 790 should only be called if debug is enabled (to invoke listToString() only when required...